src/plugin/sdl/sdl.c

```diff
@@ -238,7 +258,9 @@ static void SDL_text_init(void)
     goto tidy_ttf;
   }
 
-  pointsize = 22; // value only used if we don't resize to X_winsize
+  pointsize = 22; // values only used if we don't resize to X_winsize
```

Comment:
Why not to use vga.char_width, vga.char_height?
It would be nice if the initial window size would be
similar (or close to) in ROM and ttf modes.